guides/common/modules/proc_setting-discovery-as-the-default-PXE-boot-option.adoc

```diff
@@ -0,0 +1,25 @@
+[id="setting-discovery-as-the-default-PXE-boot-option_{context}"]
+= Setting Discovery as the default PXE boot option
+
+Set the Discovery service as the default service that boots for hosts that are not present in your current {Project} inventory.
+
+When you start an unknown host in PXE mode, {ProjectServer} or {SmartProxyServer} provides a boot menu with a default boot option.
+The boot menu has two basic options: `local` and `discovery`.
```

Comment:
```suggestion
The boot menu has two options: `local` and `discovery`.
```

Reply:
It has more than two options, but only these two are relevant in this context.